src/components/Header/SettingsToggle.tsx

```diff
@@ -54,6 +55,7 @@ const SettingsToggle = (props: SettingsToggleProps) => {
                 : ({ className: itemClassName }) => (
                     <ChromeLink {...rest} className={itemClassName} href={url} rel={rel} isBeta={isPreview}>
                       {title}
+                      {titleLabel}
```

Comment:
I believe we should add this to the dropdown item description prop: https://www.patternfly.org/components/menus/dropdown#with-item-descriptions

I know we use custom dropdown component definition, but I'd like to see if we can use the `description` prop first on the dropdown item.

Reply:
If we pass the title as a child to the dropdown item itself, we should be able to just pass the props to the ChromeLink (it should get the children prop).

But if the looks off we will have to stick with this.

Reply:
Done